lab-David/test/read-file.test.js

```diff
@@ -0,0 +1,49 @@
+//4f6e65204f6e6520
+//54776f2054776f20
+//5468726565205468
+'use strict';
+const assert = require('assert');
+const read = require('../lib/read-file');
+
+describe('read-file.js', function(){
+  describe('#readData()', function(){
+    it('should return the first 8 bytes of one.txt in hex format', function(done){
+      read.readData('one.txt', function(err, parsedData){
+        assert.ifError(err);
+        assert.strictEqual(parsedData, '4f6e65204f6e6520');
+
+        done();
+      });
+    });
+    /* it('should return an error if given a bad file', function (done) {
+      read.readData('one.txt', function(err, parsedData){
+        assert.notStrictEqual(err, null);
+        assert.strictEqual(parsedData, undefined);
+
+        done();
+      });
+    }); */
+  });
+  describe('#readFiles()', function(){
+    it('should return first 8 bytes of one.txt, two.txt, three.txt in hex format', function(done){
+      read.readFiles('one.txt', 'two.txt', 'three.txt', function(err, data1, data2, data3) {
+        assert.ifError(err);
+        assert.strictEqual(data1, '4f6e65204f6e6520');
+        assert.strictEqual(data2, '54776f2054776f20');
+        assert.strictEqual(data3, '5468726565205468');
+
+        done();
+      });
+    });
+    it('should give an err if given bad files', function (done) {
+      read.readFiles('1.txt', '2.txt', '3.txt', function(err, data1, data2, data3){
+        assert.notStrictEqual(err, null);
```

Comment:
Interesting design choice... if all three files are invalid, which `err` do we get here?